report/abstract.tex

```diff
@@ -1,5 +1,5 @@
 \textbf{With an increasing popularity of online games, the demand for resources required to deploy gaming services is rising.
-	Motivated by the shortcomings of a centralized system, a distributed architecture of the Fault-tolerant Distributed Dragon Game (FDDG) has been built on request for WantGame BV. 
+	Motivated by the shortcomings of a centralized system, a distributed architecture of the Fault-tolerant Distributed Dragon Game (FDDG) is developed on request for WantGame BV. 
 	The design of the game focuses on fault tolerance and consistency.
 	Fault tolerance is achieved by making use of a mirrored game architecture and consistency is guaranteed by letting all servers acknowledge the requests for actions of players. 
 	Through experiments, the performance of the system is measured and the limits and tradeoffs are discussed.
```

Comment:
Not everything is in active voice now. See http://gyazo.com/935b8e7aa72196b22668b4fc0759cda6